### DISCUSSION TITLE
Language pack

### DISCUSSION THREAD

**Discussion by edoned** [Mr-MIBonk/M.I.B._More-Incredible-Bash#198] [Q&A]
Hi guys, 
i kinda need your help. 
I have weird problem with my freshly installed language pack from M.I.B. 3.5.1. Spring Edition on my SW TR MHI2Q_ER_AUG22_P5092MU1329 (audi Q5).
So in my case i installed Slovenian language (slovenščina) and it worked great on the middle screen but sadly virtual cockpit doesn't accept new language. When i go for example to german language and then back to slovenian i have VC in german and middle screen in slovenian and then when i go to english and back to slovenian i have VC in english and middle screen in slovenian.
So if anyone could help that would be great. Thanks

**Comment by harman-f** [comment]
My asumption woul be, that your VC FW does not support Slovenian.

**Reply by edoned** [reply]
Hi and thanks for answer. 
Do you know maybe if the latest FW MHI2Q_ER_AUG22_P5152 would make any difference in my case? 
Thanks

**Reply by edoned** [reply]
Sorry i mean **SW** MHI2Q_ER_AUG22_P5152.

**Reply by harman-f** [reply]
That is the MMI software, which does support slovenien after "patching" language dataset.
VC is a different story.

**Reply by edoned** [reply]
ok understood. 
Thank You very much.
You guys are amazing!!!

**Reply by DipeshChunilal** [reply]
Hi all, hope all well?
need help finding the software on mibsolution:
currently running: MHI2Q_ER_AUG22_P5092 and according to mibhelper a software: MHI2Q_ER_AUG22_P5152 is available to download but cannot find it? its for an AUDI A4 2Tdi black edittion 2017 if help. the P5092 was an update i done last year through help from here.
thanks in advance

**Comment by Gasper6201** [comment]
If I may ask where di you find the slovenian language? I'd need it but can't find anything online.

**Comment by ivanzelenika95** [comment]
Hi , can someone send me link where i can find language packs , i need Croatian , Please help !
